lib/src/markdown_processor.dart

```diff
@@ -797,7 +797,6 @@ class MarkdownDocument extends md.Document {
       }
 
       bool specifiesLanguage = pre.classes.isNotEmpty;
-      pre.classes.add('prettyprint');
```

Comment:
Could you mention a bit more about how this is replaced?

Reply:
`prettyprint` was used by the pretty print library to find elements to style at runtime.

`highlight.js` looks at all code blocks with `language-XYZ` – no prettyprint needed